rust/operator-binary/src/backend/tls.rs

```diff
@@ -252,8 +305,13 @@ impl SecretBackend for TlsGenerate {
     ) -> Result<SecretContents, Self::Error> {
         let now = OffsetDateTime::now_utc();
         let not_before = now - Duration::minutes(5);
-        let not_after = now + Duration::days(1);
-        let expire_pod_after = not_after - Duration::minutes(30);
+        let autotls_cert_lifetime = selector
+            .autotls_cert_lifetime
+            .and_then(|l| l.try_into().ok());
+        let cert_lifetime = autotls_cert_lifetime.unwrap_or(self.max_cert_lifetime);
```

Comment:
That would increase the cert lifetime massively.

Reply:
This code was removed, we now default to 1d, which can be overridden by a Volume annotation, but not more than the SecretClass maxCertLifetime allows

Reply:
Implementation switched to what we have decided upon in the Arch meeting